fwoa-tools/src/binaryConverter.ts

```diff
@@ -154,43 +147,58 @@ async function convertBinaryResource(): Promise<void> {
         outputFile.file_names[folderName] = [];
       // eslint-disable-next-line security/detect-object-injection
       outputFile.file_names[folderName].push(newKey);
-      await uploadBinaryResource(newKey, results);
+      await uploadBinaryResource(newKey, results, s3Client);
       logs.write(`${new Date().toISOString()}: Updated Binary .ndjson uploaded to Export Bucket!\n`);
     }
     results = results.trimEnd();
   }
 }
 
 async function checkConfiguration(): Promise<void> {
-  await s3Client.listObjectsV2({ Bucket: exportBucketName! }).promise();
+  const s3Client: S3 = new S3({
+    region: process.env.API_AWS_REGION
+  });
+  await s3Client.listObjectsV2({ Bucket: process.env.EXPORT_BUCKET_NAME! }).promise();
   console.log('Sucessfully authenticated with S3 Export Bucket...');
-  await s3Client.listObjectsV2({ Bucket: binaryBucketName! }).promise();
+  await s3Client.listObjectsV2({ Bucket: process.env.BINARY_BUCKET_NAME! }).promise();
   console.log('Sucessfully authenticated with S3 Binary Bucket...');
 }
 
-async function startBinaryConversion(): Promise<void> {
+async function startBinaryConversion(logs: WriteStream, outputFile: ExportOutput): Promise<void> {
   console.log(`Starting Binary Resource Conversion...`);
   logs.write(`${new Date().toISOString()}: Starting Binary Resource Conversion`);
-  await convertBinaryResource();
+  await convertBinaryResource(logs, outputFile);
   // eslint-disable-next-line security/detect-non-literal-fs-filename
   writeFileSync(`${EXPORT_STATE_FILE_NAME}`, JSON.stringify(outputFile));
   logs.write(`${new Date().toISOString()}: Finished Binary Resource Conversion`);
 }
 
 (async () => {
-  await checkConfiguration();
-  console.log('successfully authenticated to all services');
-  if (!dryRun) {
-    try {
-      await startBinaryConversion();
+  // eslint-disable-next-line security/detect-non-literal-fs-filename
+  const logs = createWriteStream(`${CONVERSION_OUTPUT_LOG_FILE_PREFIX}${Date.now().toString()}.log`, {
+    flags: 'a'
+  });
+  // eslint-disable-next-line security/detect-non-literal-fs-filename
+  const outputFile = JSON.parse(readFileSync(EXPORT_STATE_FILE_NAME).toString());
+
+  if (!process.env.BINARY_BUCKET_NAME) {
+    throw new Error('BINARY_BUCKET_NAME environment variable not specified');
+  }
+  if (!process.env.EXPORT_BUCKET_NAME) {
+    throw new Error('EXPORT_BUCKET_NAME environment variable not specified');
+  }
```

Comment:
I don't think we need this since these env vars are already being checked by `await checkConfiguration`